code/go/0chain.net/smartcontract/dbs/event/error.go

```diff
@@ -0,0 +1,18 @@
+package event
+
+import "gorm.io/gorm"
+
+type Error struct {
+	gorm.Model
+	TransactionID string
+	Error         string
+}
+
+func (edb *EventDb) addError(err Error) error {
+	return edb.Store.Get().Create(&err).Error
+}
+
+func (edb *EventDb) GetErrorByTransactionHash(transactionID string) ([]Error, error) {
+	var err []Error
```

Comment:
probably a better variable name for error data from db
and not err. err is generally used for go based errors.

Reply:
done